### ISSUE TITLE
AutoComplete suggestionsAsTable cause an a11y problem

### ISSUE THREAD

**Issue by EugenFischer** [primefaces/primefaces#12918]
### Describe the bug

Rendered suggestion table does include table rows with role="option", but the surrounded table do not include the role="listbox".
This violate the rule "Certain ARIA roles must be contained by particular parent elements"

### Reproducer

1. Navigate to https://www.primefaces.org/showcase/ui/input/autoComplete.xhtml
2. execute autocompletion for Custom Content example
3. inspect the rendered suggection table

Currently rendered
```
<table class="ui-autocomplete-items ui-autocomplete-table ui-widget-content ui-widget ui-corner-all ui-helper-reset">
...
</table>
```

### Expected behavior

```
<table class="ui-autocomplete-items ui-autocomplete-table ui-widget-content ui-widget ui-corner-all ui-helper-reset" 
role="listbox">
...
</table>
```

### PrimeFaces edition

Community

### PrimeFaces version

13.0.10, 14.0.6

### JSF implementation

Mojarra

### JSF version

4.0.1

### Java version

17

### Browser(s)

Edge

**Comment by EugenFischer** [comment]
I've saw the **autocomplete.js** does include the function **findWrapperTag** which (like i understand) should care about.

Suggestion : may it's easier to move this into **AutoCompleteRenderer** -> **encodeSuggestionsAsTable**

**Comment by melloware** [comment]
Thanks fixing for 14.0.8

**Comment by EugenFischer** [comment]
awesome thx.
could you please provide the fix also for version 13?

**Comment by tandraschko** [comment]
we can't; please read: https://github.com/primefaces/primefaces?tab=readme-ov-file#community--lts--pro
